modules/node_modules/@colony/purser-trezor/staticMethods.js

```diff
@@ -308,7 +309,8 @@ export const signMessage = async ({
           derivationPathNormalizer(derivationPath),
           true,
         ).toPathArray(),
-        message,
+        // $FlowFixMe need Buffer types
+        message: Buffer.from(toSign).toString(HEX_HASH_TYPE),
```

Comment:
Trezor seems to work:

![screenshot from 2019-01-20 10-59-29](https://user-images.githubusercontent.com/1193222/51437144-a7fa7700-1ca2-11e9-959d-6a5d786e211b.png)
Produces:
![screenshot from 2019-01-20 10-59-39](https://user-images.githubusercontent.com/1193222/51437145-a8930d80-1ca2-11e9-9835-5bef49ab36df.png)